### PR TITLE
[zh] Sync create-external-load-balancer.md

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@fujitsu.com>

Work for https://github.com/kubernetes/website/issues/33477 tasks-11
(The part-2 of tasks-11: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md)
/assign @tengqm
PTAL, thanks.

### CODE REVIEW

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 1]
```diff
@@ -13,62 +13,64 @@ weight: 80
...
-that sends traffic to the correct port on your cluster nodes
+When creating a {{< glossary_tooltip text="Service" term_id="service" >}}, you have
+the option of automatically creating a cloud load balancer. This provides an
+externally-accessible IP address that sends traffic to the correct port on your cluster
+nodes,
 _provided your cluster runs in a supported environment and is configured with
 the correct cloud load balancer provider package_.
 -->
-创建服务时，你可以选择自动创建云网络负载均衡器。这提供了一个外部可访问的 IP 地址，
+创建 {{< glossary_tooltip text="服务" term_id="service" >}} 时，你可以选择自动创建云网络负载均衡器。这提供了一个外部可访问的 IP 地址，
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:28]:
```suggestion
创建 {{< glossary_tooltip text="服务" term_id="service" >}} 时，你可以选择自动创建云网络负载均衡器。
负载均衡器提供外部可访问的 IP 地址，
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 2]
```diff
@@ -13,62 +13,64 @@ weight: 80
...
 可将流量分配到集群节点上的正确端口上
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:29]:
```suggestion
可将流量发送到集群节点上的正确端口上
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 3]
```diff
@@ -13,62 +13,64 @@ weight: 80
...
-（ _假设集群在支持的环境中运行，并配置了正确的云负载平衡器提供商包_）。
+（ _假设集群在支持的环境中运行，并配置了正确的云负载均衡器供应商包_）。
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:30]:
```suggestion
（ **假设集群在支持的环境中运行，并配置了正确的云负载均衡器驱动包**）。
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 4]
```diff
@@ -13,62 +13,64 @@ weight: 80
...
-For information on provisioning and using an Ingress resource that can give
-services externally-reachable URLs, load balance the traffic, terminate SSL etc.,
-please check the [Ingress](/docs/concepts/services-networking/ingress/)
+You can also use an {{< glossary_tooltip term_id="ingress" >}} in place of Service.
+For more information, check the [Ingress](/docs/concepts/services-networking/ingress/)
 documentation.
 -->
-有关如何配置和使用 Ingress 资源为服务提供外部可访问的 URL、负载均衡流量、终止 SSL 等功能，
-请查看 [Ingress](/zh/docs/concepts/services-networking/ingress/) 文档。
+你还可以使用 {{< glossary_tooltip text="Ingress" term_id="ingress" >}} 代替服务。
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:37]: Service 这类词，首字母大写的时候，指代的是资源类型名称，不要翻译。

```suggestion
你还可以使用 {{< glossary_tooltip text="Ingress" term_id="ingress" >}} 代替 Service。
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 5]
```diff
@@ -13,62 +13,64 @@ weight: 80
...
+### Create a Service from a manifest
 
 To create an external load balancer, add the following line to your
-[service configuration file](/docs/concepts/services-networking/service/#loadbalancer):
+Service manifest:
 -->
-## 配置文件
+## 创建服务
+
+### 从清单文件创建服务
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:62]:
```suggestion
### 基于清单文件创建服务
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 6]
```diff
@@ -13,62 +13,64 @@ weight: 80
...
 
-要创建外部负载均衡器，请将以下内容添加到
-[服务配置文件](/zh/docs/concepts/services-networking/service/#loadbalancer)：
+要创建外部负载均衡器，请将以下内容添加到你的服务清单文件：
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:64]:
```suggestion
要创建外部负载均衡器，请将以下内容添加到你的 Service 清单文件：
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 7]
```diff
@@ -85,29 +87,31 @@ spec:
...
 
 <!--
-## Using kubectl
+### Create a Service using kubectl
 
 You can alternatively create the service with the `kubectl expose` command and
 its `--type=LoadBalancer` flag:
 -->
-## 使用 kubectl
+### 使用 kubectl 创建服务
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:95]:
```suggestion
### 使用 kubectl 创建 Service
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 8]
```diff
@@ -166,52 +174,52 @@ minikube service example-service --url
...
-service spec (supported in GCE/Google Kubernetes Engine environments):
+By default, the source IP seen in the target container is *not the original
+source IP* of the client. To enable preservation of the client IP, the following
+fields can be configured in the `.spec` of the Service:
 -->
 ## 保留客户端源 IP
 
-由于此功能的实现，目标容器中看到的源 IP 将 *不是客户端的原始源 IP*。
-要启用保留客户端 IP，可以在服务的 spec 中配置以下字段（支持 GCE/Google Kubernetes Engine  环境）：
+默认情况下，目标容器中看到的源 IP 将 *不是客户端的原始源 IP*。
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:183]:
```suggestion
默认情况下，目标容器中看到的源 IP 将**不是客户端的原始源 IP**。
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 9]
```diff
@@ -166,52 +174,52 @@ minikube service example-service --url
...
+  IP and may cause a second hop to another node, but should have good overall
+  load-spreading. `Local` preserves the client source IP and avoids a second hop
+  for LoadBalancer and NodePort type Services, but risks potentially imbalanced
+  traffic spreading.
 -->
-* `service.spec.externalTrafficPolicy` - 表示此服务是否希望将外部流量路由到节点本地或集群范围的端点。
-  有两个可用选项：Cluster（默认）和 Local。
-  Cluster 隐藏了客户端源 IP，可能导致第二跳到另一个节点，但具有良好的整体负载分布。
-  Local 保留客户端源 IP 并避免 LoadBalancer 和 NodePort 类型服务的第二跳，
+* `.spec.externalTrafficPolicy` - 表示此服务是否希望将外部流量路由到节点本地或集群范围的端点。
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:195]:
```suggestion
* `.spec.externalTrafficPolicy` - 表示此 Service 是否希望将外部流量路由到节点本地或集群范围的端点。
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 10]
```diff
@@ -166,52 +174,52 @@ minikube service example-service --url
...
+  to `Local`.
 -->
-* `service.spec.healthCheckNodePort` - 指定服务的 healthcheck nodePort（数字端口号）。
-  如果未指定 `healthCheckNodePort`，服务控制器从集群的 NodePort 范围内分配一个端口。
+* `.spec.healthCheckNodePort` - 指定服务的 healthcheck nodePort（数字端口号）。
+  如果你未指定 `healthCheckNodePort`，服务控制器从集群的 NodePort 范围内分配一个端口。
   你可以通过设置 API 服务器的命令行选项 `--service-node-port-range` 来配置上述范围。
-  它将会使用用户指定的 `healthCheckNodePort` 值（如果被客户端指定）。
-  仅当 `type`  设置为 LoadBalancer 并且 `externalTrafficPolicy` 设置为 Local 时才生效。
+  在服务 `type` 设置为 LoadBalancer 并且 `externalTrafficPolicy` 设置为 Local 的前提下，
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:215]:
```suggestion
  在服务 `type` 设置为 LoadBalancer 并且 `externalTrafficPolicy` 设置为 Local 时，
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 11]
```diff
@@ -166,52 +174,52 @@ minikube service example-service --url
...
+  服务将会使用用户指定的 `healthCheckNodePort` 值（如果你指定了它）。
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:216]:
```suggestion
  Service 将会使用用户指定的 `healthCheckNodePort` 值（如果你指定了它）。
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 12]
```diff
@@ -258,63 +297,35 @@ service controller crashing.
...
-## Caveats and Limitations when preserving source IPs
-
-GCE/AWS load balancers do not provide weights for their target pools. This was not an issue with the old LB
-kube-proxy rules which would correctly balance across all endpoints.
--->
-## 保留源 IP 时的注意事项和限制
-
-GCE/AWS 负载均衡器不为其目标池提供权重。
-对于旧的 LB kube-proxy 规则来说，这不是一个问题，它可以在所有端点之间正确平衡。
+`type` 等于  ClusterIP，并通过使用托管了相关 Kubernetes Pod 节点的条目对负载均衡器
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:316]:
```suggestion
`type` 设置为 ClusterIP，并通过使用托管了相关 Kubernetes Pod 的节点作为条目对负载均衡器
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 13]
```diff
@@ -258,63 +297,35 @@ service controller crashing.
...
-
-Internal pod to pod traffic should behave similar to ClusterIP services, with equal probability across all pods.
--->
-一旦外部负载平衡器提供权重，就可以将此功能添加到 LB 编程路径中。
-*未来工作：1.4 版本不提供权重支持，但可能会在将来版本中添加*
-
-内部 Pod 到 Pod 的流量应该与 ClusterIP 服务类似，所有 Pod 的概率相同。
+* 阅读[服务](/zh/docs/concepts/services-networking/service/)
+* 阅读 [Ingress](/zh/docs/concepts/services-networking/ingress/)
+* 阅读[使用服务连接应用](/zh/docs/concepts/services-networking/connect-applications-service/)
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:330]:
```suggestion
* 阅读[使用 Service 连接到应用](/zh/docs/concepts/services-networking/connect-applications-service/)
```

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 4 above)
Comment (fenggw-fnst) [reply content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:37]: 谢谢告知，以后会注意。

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 14]
```diff
@@ -13,62 +13,64 @@ weight: 80
...
+## Create a Service
+
+### Create a Service from a manifest
 
 To create an external load balancer, add the following line to your
-[service configuration file](/docs/concepts/services-networking/service/#loadbalancer):
+Service manifest:
 -->
-## 配置文件
+## 创建服务
```
Comment (Sea-n) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:60]: Please add a `{#anchor}` to each title.

Ref: [Localization Guild](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E5%85%B3%E4%BA%8E%E9%93%BE%E6%8E%A5)

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 15]
```diff
@@ -166,52 +174,52 @@ minikube service example-service --url
...
+  to `Local`.
 -->
-* `service.spec.healthCheckNodePort` - 指定服务的 healthcheck nodePort（数字端口号）。
-  如果未指定 `healthCheckNodePort`，服务控制器从集群的 NodePort 范围内分配一个端口。
+* `.spec.healthCheckNodePort` - 指定服务的 healthcheck nodePort（数字端口号）。
+  如果你未指定 `healthCheckNodePort`，服务控制器从集群的 NodePort 范围内分配一个端口。
   你可以通过设置 API 服务器的命令行选项 `--service-node-port-range` 来配置上述范围。
-  它将会使用用户指定的 `healthCheckNodePort` 值（如果被客户端指定）。
-  仅当 `type`  设置为 LoadBalancer 并且 `externalTrafficPolicy` 设置为 Local 时才生效。
+  在服务 `type` 设置为 LoadBalancer 并且 `externalTrafficPolicy` 设置为 Local 时，
```
Comment (Sea-n) [inline content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md:215]: nit: `Local` should be `` `Local` `` according to English content.

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 14 above)
Comment (fenggw-fnst) [reply content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md]: Thanks for the review, added anchor as suggested.

File: content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 15 above)
Comment (fenggw-fnst) [reply content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md]: Done, thanks.

**Pull request merged**